testsuite/INSTALL

```diff
@@ -84,9 +84,9 @@ From sources
 
 From the top directory of the checked out sources:
 
-  ./testsuite/MDAnalysisTests/mda_nosetests --exe -v --processes=4 --process-timeout=120 
+  pytest /testsuite/MDAnalysisTests/ -n=3
```

Comment:
please add a note that the `pytest-xdist` package should be installed to run tests in parallel.